.buildkite/nightly-benchmarks/tests/nightly-tests.json

```diff
@@ -22,31 +23,143 @@
         "trt_server_parameters": {
             "model_type": "llama",
             "model_dtype": "float16",
-            "max_batch_size": 256,
+            "max_batch_size": 2048,
             "max_input_len": 4096,
-            "max_output_len": 4096,
-            "trt_llm_version": "r24.04"
+            "max_seq_len": 6144,
+            "max_num_tokens": 16384,
+            "trt_llm_version": "v0.11.0"
         },
         "trt_client_parameters": {
             "endpoint": "/v2/models/ensemble/generate_stream"
+        }, 
+        "vllm_055_server_parameters": {
+            "disable_log_stats": "",
+            "disable_log_requests": "",
+            "gpu_memory_utilization": 0.95,
+            "num_scheduler_steps": 10,
```

Comment:
BTW After you adjust `num_scheduler_steps`, although the throughput has increased, both Median TTFT and Median ITL will **increase significantly**, meaning the real throughput hasn't actually improved. Here, real throughput refers to the throughput for online scenarios where Median TTFT and Median ITL meet **certain latency conditions**.

Reply:
We are aware of the series of issues of increasing `num_scheduler_steps`, like bumping delay of tokens & increased TTFT. These issues are not a fundamental consequence of multi-step scheduler and can be fixed (at least from what I heard), but the throughput improvement is fundamental. So we choose to showcase throughput in the main results and still provide other results for people to check.

Being transparent is difficult. Every people want the benchmark to lean towards them. This benchmark is not perfect for now (I have also received issues on TRT-LLM recently about the dtype issue) but I believe open-source benchmark is the right way to go.

Again, could you please reach out to me via slack? It's much easier for me to see it.

Reply:
All metrics will be generated in the results. This PR does not do visualizations (though we include example visualization scripts so that people can develop their own visualization scripts easier)